multiaccounts/accounts/database.go

```diff
@@ -36,6 +73,43 @@ type Account struct {
 	Color     string         `json:"color"`
 }
 
+type Settings struct {
+	Address                common.Address      `json:"address"`
+	ChaosMode              *bool               `json:"chaos-mode,omitempty"`
+	Currency               *string `json:currency,omitempty`
+	CustomBootnodes        *json.RawMessage `json:custom-boot-nodes,omitempty`
+	CustomBootnodesEnabled *json.RawMessage `json:custom-boot-nodes-enabled,omitempty`
+	DappsAddress           common.Address `json:dapps-address`
+	EIP1581Address         common.Address `json:eip1581-address`
+	Fleet                  *string `json:fleet,omitempty`
+	HideHomeTooltip        *bool `json:hide-home-tooltip,omitempty`
+	InstallationID         string `json:installation-id`
+	KeyUID                 string `json:key-uid`
+	KeycardInstanceUID     *string `json:keycard-instance-uid,omitempty`
+	KeycardPAiredOn        *time.Time `json:keycard-paired-on,omitempty`
+	KeycardPairing         *string `json:keycard-pairing,omitempty`
+	LatestDerivedPath      uint `json:last-derived-path`
+	LogLevel               *string `json:log-level,omitempty`
+	Mnemonic               *string `json:mnemonic,omitempty`
+	Name                   string `json:name,omitempty`
+	NotificationsEnabled   *bool `json:notification-enabled,omitempty`
+	PhotoPath              string `json:photo-path`
+	PinnedMailserver       *json.RawMessage `json:pinned-mailserver,omitempty`
+	PreferredName          *string `json:preferred-name,omitempty`
+	PreviewPrivacy         bool `json:preview-privacy`
+	PublicKey              string `json:public-key`
+	RememberSyncingChoice  *bool `json:remember-syncing-choice,omitempty`
+	ShowName               *bool `json:show-name,omitempty`
+	SigningPhrase          string `json:signing-phrase`
+	StickerPacksInstalled  *json.RawMessage `json:stickers-packs-installed,omitempty`
+	StickersRecentStickers *json.RawMessage `json:stickers-recent-stickers,omitempty`
+	SyncingOnMobileNetwork *bool `json:syncing-on-mobile-network,omitempty`
+	Usernames              *[]string `json:usernames,omitempty`
+	WalletRootAddress      common.Address `json:wallet-root-address,omitempty`
+	WalletSetupPassed      *bool `json:wallet-setup-passed,omitempty`
+	WalletVisibleTokens    *[]string `json:wallet-visible-tokens,omitempty`
```

Comment:
Are we sure this is the correct syntax for naming fields?

Reply:
yes I want to avoid having to rename all fields on status-react side